src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -356,11 +360,40 @@ $ pachctl migrate --from 1.4.8 --to 1.5.0
 	migrate.Flags().StringVar(&to, "to", "", "The version of Pachyderm to migrate to.  If not specified, pachctl will use its own version.")
 	migrate.Flags().StringVar(&namespace, "namespace", "default", "The kubernetes namespace under which Pachyderm is deployed.")
 
+	completion := &cobra.Command{
+		Use:   "completion",
+		Short: "Install bash completion code.",
+		Long:  "Install bash completion code.",
+		Run: cmdutil.RunFixedArgs(0, func(args []string) (retErr error) {
+			if len(args) != 0 {
```

Comment:
This seems redundant since `RunFixedArgs` already checks for the argument count.

Reply:
Err, yup it very much is, I copied this over from the other PR and forgot to change it.